model_compression_toolkit/target_platform_capabilities/target_platform/op_quantization_config.py

```diff
@@ -299,7 +299,12 @@ def clone_and_map_weights_attr_keys(self, layer_attrs_mapping: Union[Dict[str, s
         """
         qc_options = copy.deepcopy(self)
 
-        for qc in qc_options.quantization_config_list:
+        # create a list of unique qc options to update
+        cfgs_to_update = qc_options.quantization_config_list
+        if not any(qc_options.base_config is cfg for cfg in cfgs_to_update):
```

Comment:
\# Extract the list of existing quantization configurations from qc_options

\# Check if the base_config is already included in the quantization configuration list
\# If not, add base_config to the list of configurations to update